cloudflare.go

```diff
@@ -48,6 +48,8 @@ type API struct {
 	logger            Logger
 }
 
+var _ APIV4 = &API{}
```

Comment:
This on its own doesn't really provide much context as to the intention. Does `CloudflareAPIv4` work and meet your original goal of including "cloudflare" in the interface name? 

```suggestion
var _ CloudflareAPIv4 = &API{}
```

Reply:
Yeah that works, though I suspect the linter may complain